pkgs/development/libraries/qt-4.x/4.8/default.nix

```diff
@@ -164,7 +164,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ perl pkgconfig which ];
 
-  enableParallelBuilding = false;
+  enableParallelBuilding = stdenv.isDarwin;
```

Comment:
Not sure if we want to enable this since it's only for darwin; I haven't tested to see if it even improves build time.

Reply:
https://github.com/NixOS/nixpkgs/commit/da9115128cd5a200a82fc1d30c6ae1849fae2fb6, might be safer to keep it disabled.

Reply:
I'm pretty it still only used one build process with it enabled now that I think about it.
Reverted. Should I rebase? apparently I the original commit has a formatting change

Reply:
yeah, I'll merge and backport this if you squash the commits

Reply:
👍